pkg/golinters/importas/importas.go

```diff
@@ -51,11 +51,15 @@ func New(settings *config.ImportAsSettings) *goanalysis.Linter {
 				uniqPackages[a.Pkg] = a
 			}
 
-			// skip the duplication check when the alias is a regular expression replacement pattern (ie. contains `$`).
-			if v, ok := uniqAliases[a.Alias]; ok && !strings.Contains(a.Alias, "$") {
-				lintCtx.Log.Errorf("invalid configuration, multiple packages with the same alias: alias=%s packages=[%s,%s]", a.Alias, a.Pkg, v.Pkg)
-			} else {
-				uniqAliases[a.Alias] = a
+			// Ignore duplication for empty aliases.
+			if a.Alias != "" {
+				// skip the duplication check when the alias is a regular expression replacement pattern (ie. contains `$`).
+				v, ok := uniqAliases[a.Alias]
+				if ok && !strings.Contains(a.Alias, "$") {
+					lintCtx.Log.Errorf("invalid configuration, multiple packages with the same alias: alias=%s packages=[%s,%s]", a.Alias, a.Pkg, v.Pkg)
+				} else {
```

Comment:
An alternative solution (will read from map and never get the value, but this reduces the indentation).

```go
			if v, ok := uniqAliases[a.Alias]; ok && !strings.Contains(a.Alias, "$") {
 				lintCtx.Log.Errorf("invalid configuration, multiple packages with the same alias: alias=%s packages=[%s,%s]", a.Alias, a.Pkg, v.Pkg)
 			} else if a.Alias != "" {
 				uniqAliases[a.Alias] = a
 			}
```